src/ui/server/controllers/insurance/business/broker/map-submitted-data/index.ts

```diff
@@ -0,0 +1,14 @@
+import { RequestBody } from '../../../../../../types';
+
+/**
+ * maps broker formBody and removes _crsf
+ * @param {RequestBody} formBody
+ * @returns {Object} populatedData
+ */
+const mapSubmittedData = (formBody: RequestBody): object => {
```

Comment:
Doesn't seem that we need this function at all for the broker section? The only thing that's happening is `_csrf` is stripped, but this already gets stripped in sanitise-data.